backend/src/seeds/transactionLink/index.ts

```diff
@@ -0,0 +1,52 @@
+import { TransactionLinkInterface } from './TransactionLinkInterface'
+
+export const transactionLinks: TransactionLinkInterface[] = [
+  {
+    email: 'bibi@bloxberg.de',
+    amount: 19.99,
```

Comment:
use the value to construct meaningful cases. `19.99` is an float test - but there might be other useful things to test, like decay rounding errors etc.